operator/tests/central/basic-central/80-assert-non-openshift.yaml

```diff
@@ -0,0 +1,52 @@
+# Make kuttl ignore this file if running against openshift.
+apiVersion: kuttl.dev/v1beta1
+kind: TestFile
+testRunSelector:
+  matchLabels:
+    openshift: "false"
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+collectors:
+- type: pod
+  selector: app=central
+  tail: -1
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: central
+spec:
+  template:
+    spec:
+      containers:
+        - name: central
+          env:
+            - name: GOMEMLIMIT
+              valueFrom:
+                resourceFieldRef:
+                  resource: limits.memory
+            - name: GOMAXPROCS
+              valueFrom:
+                resourceFieldRef:
+                  resource: limits.cpu
+            - name: ROX_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.namespace
+            - name: ROX_TELEMETRY_ENDPOINT
+              value: https://telemetry.endpoint
+            - name: ROX_TELEMETRY_STORAGE_KEY_V1
+              value: my-api-key
+            - name: ROX_OFFLINE_MODE
+              value: "false"
```

Comment:
Is this needed since `ROX_OFFLINE_MODE` seems to be false by default ?

Reply:
Not sure if that's what you're asking, but yes, it's needed, otherwise the test would fail because of very rudimental support for arrays in kuttl.